nidx/nidx_vector/src/searcher.rs

```diff
@@ -667,6 +669,7 @@ mod tests {
             split: "".to_string(),
             repeated_in_field: false,
             metadata: None,
+            ..Default::default()
```

Comment:
It would be nice to have a test somewhere that uses the `text` field to make sure it works.